src/ContextFreeTasks.Shared/ContextFreeTask.cs

```diff
@@ -11,5 +11,6 @@ public struct ContextFreeTask
         public Task Task => _task ?? FromResult(default(object));
         internal ContextFreeTask(Task t) => _task = t;
         public ContextFreeTaskAwaiter GetAwaiter() => new ContextFreeTaskAwaiter(Task);
+        public void Wait() => _task?.Wait();
```

Comment:
in my opinion, naming of 'ContextFree' includes the meaning 'This can blocking-wait safely'.
Task and ValueTask should not use blocking-wait, but ContextFreeTask doesn't.
It is possible but, of course I think that is better not to use.